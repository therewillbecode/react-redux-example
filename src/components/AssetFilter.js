import React from "react";
import { Dropdown, Menu } from "semantic-ui-react";

const options = [
  { key: 1, text: "All", value: "all" },
  { key: 2, text: "Received", value: "received" },
  { key: 3, text: "Dispatched", value: "dispatched" }
];

const AssetFilter = ({ handleChange }) => (
  <Dropdown
    text="Filter Assets"
    button
    icon="filter"
    options={options}
    onChange={handleChange}
    className="icon"
  />
);

export default AssetFilter;
