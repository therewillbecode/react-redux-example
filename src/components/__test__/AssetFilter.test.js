/* eslint-disable */
import React from "react";
import renderer from "react-test-renderer";

import AssetFilter from "../AssetFilter";

describe("AssetFilter", () => {
  beforeEach(() => {});
  it("renders correctly", () => {
    const tree = renderer.create(<AssetFilter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
