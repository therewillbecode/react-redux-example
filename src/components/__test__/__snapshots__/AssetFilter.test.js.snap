// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetFilter renders correctly 1`] = `
<div
  style={
    Object {
      "padding": "2em",
    }
  }
>
  <div
    aria-busy={undefined}
    aria-disabled={undefined}
    aria-expanded={false}
    aria-multiselectable={undefined}
    className="ui button dropdown icon"
    onBlur={[Function]}
    onChange={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onMouseDown={[Function]}
    role="listbox"
    tabIndex={0}
  >
    <div
      aria-live="polite"
      className="text"
      role="alert"
    >
      Filter Assets
    </div>
    <i
      aria-hidden="true"
      className="filter icon"
      onClick={[Function]}
    />
    <div
      className="menu transition"
    >
      <div
        aria-checked={false}
        aria-disabled={undefined}
        aria-selected={true}
        className="selected item"
        onClick={[Function]}
        role="option"
        style={
          Object {
            "pointerEvents": "all",
          }
        }
      >
        <span
          className="text"
        >
          All
        </span>
      </div>
      <div
        aria-checked={false}
        aria-disabled={undefined}
        aria-selected={false}
        className="item"
        onClick={[Function]}
        role="option"
        style={
          Object {
            "pointerEvents": "all",
          }
        }
      >
        <span
          className="text"
        >
          Received
        </span>
      </div>
      <div
        aria-checked={false}
        aria-disabled={undefined}
        aria-selected={false}
        className="item"
        onClick={[Function]}
        role="option"
        style={
          Object {
            "pointerEvents": "all",
          }
        }
      >
        <span
          className="text"
        >
          Dispatched
        </span>
      </div>
    </div>
  </div>
  <div
    className="ui labeled input"
  >
    <div
      className="ui label label"
      onClick={[Function]}
    >
      search by name
    </div>
    <input
      disabled={undefined}
      name="name"
      onChange={[Function]}
      placeholder="Name"
      tabIndex={undefined}
      type="text"
      value={undefined}
    />
  </div>
</div>
`;
